src/main/java/stirling/software/SPDF/config/AppConfig.java

```diff
@@ -135,4 +139,20 @@ public Predicate<Path> processOnlyFiles() {
             }
         };
     }
+
+    @Bean
```

Comment:
no need to make a new settings file requirement here.
This code also Requires the entry
currently the PR fails with
``` Then the response content type should be "application/octet-stream"               # features/steps/step_definitions.py:266
      Assertion Failed: Expected application/octet-stream but got application/json. Response content: b'{"timestamp":"2024-09-03T09:09:03.423+00:00","status":500,"error":"Internal Server Error","exception":"java.lang.NullPointerException","trace":"java.lang.NullPointerException: Cannot invoke \\"stirling.software.SPDF.config.memoryConfig$MemorySettings.getRamThresholdGB()\\" because the return value of \\"stirling.software.SPDF.config.memoryConfig.getMemory()\\" is null\\n\\tat stirling.software.SPDF.utils.memoryUtils.shouldUseFileBasedStorage(memoryUtils.java:17)\\n\\tat 
 ```
      
      
      
 Please add this to the actual settings.yml instead
      https://github.com/Stirling-Tools/Stirling-PDF/blob/main/src/main/resources/settings.yml.template
      and
      https://github.com/Stirling-Tools/Stirling-PDF/blob/main/src/main/java/stirling/software/SPDF/model/ApplicationProperties.java